content/blog/2023-11-23-announcing-axum-0-7-0.md

```diff
@@ -0,0 +1,184 @@
+---
+date: "2023-11-23"
+title: "Announcing axum 0.7.0"
+description: "November 23, 2022"
+---
+
+Today, we're happy to announce [`axum`] version 0.7. `axum` is an ergonomic
+and modular web framework built with [`tokio`], [`tower`], and [`hyper`].
+
+This also includes new major versions of [`axum-core`], [`axum-extra`], and
+[`axum-macros`].
+
+## `hyper` 1.0 support
+
+The headline feature of `axum` 0.7 is support for `hyper` 1.0. `hyper` is a
+foundational library for much of the networking ecosystem in Rust and finally
+having a stable API is a big milestone.
+
+`hyper` 1.0 comes with a big shuffling of the APIs. The previous low level APIs
+(found in [`hyper::server::conn`]) is what has been stabilized whereas the high
+level APIs (such as [`hyper::Server`]) has been removed.
+
+The plan is to add the high level APIs to a new crate called [`hyper-util`].
+There we can build out the APIs without worrying too much about stability
+guarantees and backwards compatibility. When something is ready for
+stabilization it can be moved into `hyper`.
```

Comment:
The plans on the hyper side don't seem *that* relevant for this blog post, if this is mentioned over there already maybe link to the hyper 1.0 announcement and cut down this paragraph a bit? (in general it would be nice to link to that article somewhere)

Reply:
I've added a link to Sean's announcement post but I still think this paragraph is good to have since it provides context for hyper-util, without requiring readers to read another post.